recipes/neuraloperator/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "neuraloperator" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/neuraloperator-{{ version }}.tar.gz
+  sha256: caac44f3f83d58cd630fff249e465ad87ee8241e17daa75f0f1d6000c1989b84
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=64
+    - pip
+  run:
+    - python >={{ python_min }}
+    - wandb
+    - ruamel_yaml
+    - configmypy
+    - tensorly
+    - tensorly-torch
+    - torch-harmonics ==0.7.3
+    - matplotlib-base
+    - numpy >=1.25
+    - opt-einsum
+    - h5py
+    - zarr
+
+test:
+  # imports:
+  #   - neuralop
```

Comment:
Why you are not doing the import test?

Reply:
I think the import test was not working correctly (see https://github.com/conda-forge/staged-recipes/pull/28667/commits/61114c5177de8543ee9883ccefcfcdf57fa96e4a), hence I have left it commented. I will discuss this with the package authors and suggest a couple of changes.

Reply:
Plaease re-add it here so we can see the logs and how it fails.